src/main/java/blusunrize/immersiveengineering/common/blocks/IEBaseBlock.java

```diff
@@ -279,19 +293,19 @@ public ResourceLocation createRegistryName()
 	}
 
 	@Override
+	@SuppressWarnings("deprecation")
 	public boolean onBlockActivated(BlockState state, World world, BlockPos pos, PlayerEntity player, Hand hand,
-									Direction side, float hitX, float hitY, float hitZ)
+																	BlockRayTraceResult hit)
```

Comment:
The indenting is strange here, what do you use for formatting? There's an IntelliJ formatting XML for IE available [here](https://gist.github.com/BluSunrize/5c05af8f29de9581426551d9b29b1809).

Reply:
Thanks for the hint, imported the XML and corrected the indent (f0088b34).